#!/usr/bin/env python3
PACKAGE = "obstacle_publisher"
import rospkg
import yaml, os
from dynamic_reconfigure.parameter_generator_catkin import *

ros_path = rospkg.RosPack().get_path('obstacle_publisher')


gen = ParameterGenerator()
gen.add("speed_scaler", double_t, 0, "Speed of opponent: Global wpnts times speed scaler", 0.0, 0.1, 1.25)
gen.add("ampl_sin1", double_t, 0, "Amplitude of frist sinus", 0.0, 0.1, 1.0)
gen.add("ampl_sin2", double_t, 0, "Amplitude of second sinus", 0.0, 0.1, 1.0)
gen.add("phase_sin1", double_t, 0, "Phase of first sinus", 4, 0.5, 8)
gen.add("phase_sin2", double_t, 0, "Phase of second sinus", 3, 0.5, 8)

exit(gen.generate(PACKAGE, "obs_dynaimc_publisher", "dyn_obs_publisher"))